packages/react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -590,7 +590,8 @@ class Dropdown extends AutoControlledComponent<Extendable<DropdownProps>, Dropdo
               key: (item as any).header,
             }),
         },
-        overrideProps: () => this.handleItemOverrides(item, index, getItemProps),
+        overrideProps: (predefinedProps: DropdownItemProps) =>
+          this.handleItemOverrides(predefinedProps, item, index, getItemProps),
```

Comment:
Please consider to refactor to this:
```jsx
overrideProps: this.handleItemOverrides(item, index, getItemProps),
```
```tsx
  private handleItemOverrides = (
    item: ShorthandValue,
    index: number,
    getItemProps: (options: GetItemPropsOptions<ShorthandValue>) => any),
  => (predefinedProps: DropdownItemProps) => any,
```

Reply:
good idea! will update all callbacks similar to this one. and will add unit tests for them, thanks!